docs/source/services/services.rst

```diff
@@ -85,10 +85,6 @@ Available configuration options
         +----------------+-------------------------------------------------------------------+-----------------+
         | port           | Port the REST Service is listening on                             |  **Required**   |
         +----------------+-------------------------------------------------------------------+-----------------+
-        | secret_key     | 32 byte secret key to generate and validate JWTs                  |  **Required**   |
```

Comment:
I have removed outdated documentation of the REST API. Those parts have been changed in #250, but the documentation has not been updated.